packages/sync-actions/src/utils/common-actions.js

```diff
@@ -42,7 +61,7 @@ export function buildBaseAttributesActions({ actions, diff, oldObj, newObj }) {
       const patched = diffpatcher.patch(clone(before), delta)
       return { action: item.action, [actionKey]: patched }
     })
-    .filter(action => action)
+    .filter(action => !isNil(action))
```

Comment:
Not part of the PR's purpose, but I chose to be explicit here

Reply:
Are you sure that filters out the exact same things?

Reply:
We seem to always return `undefined` above whenever we aren't supposed to generate an update-action

Reply:
no we don't. we return `undefined` above